CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
+### Notable changes
```

Comment:
Huh. This is weird. This note should be in the CHANGELOG on `master`, but it is not. I suspect it got removed somehow by a bad merge on someone else's PR. Regardless, thanks for re-adding it